### ISSUE TITLE
Wrong device in graph - Tensorboard SummaryWriter 

### ISSUE THREAD

**Issue by Spenhouet** [pytorch/pytorch#23749]
## 🐛 Bug

The TensorBoard graph always displays "CPU:0" as device placement. No GPU is shown.

## To Reproduce

1. Use the `torch.utils.tensorboard.SummaryWriter`
1. Move the `model` and input data `images` to the GPU (`cuda:0`)
1. Save the graph with `writer.add_graph(model, input_to_model=images)`
1. Open TensorBoard and look at the device placement

My implementation is simply the code of the tutorial:
https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py

## Expected behavior

TensorBoard shows the correct device.

## Environment

PyTorch version: 1.1.0
Is debug build: No
CUDA used to build PyTorch: 10.0

OS: Microsoft Windows 10 Pro
GCC version: Could not collect
CMake version: Could not collect

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: 10.0.130
GPU models and configuration: GPU 0: GeForce GTX 1080
Nvidia driver version: 431.60
cuDNN version: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin\cudnn64_7.dll

Versions of relevant libraries:
[pip3] numpy==1.17.0
[pip3] torch==1.1.0
[pip3] torchvision==0.3.0
[pip3] tensorflow==1.14.0
[conda] Could not collect

## Additional context

This does not work on my local PC nor on a GPU cluster.
I know that this feature is marked as `Experimental` but I don't know if this is a bug or if it is simply not implemented as of yet.

**Comment by lanpa** [comment]
Yup, this is a placeholder.https://github.com/pytorch/pytorch/blob/bb4f380f35cc674b69243b75d018e8d62a0ed113/torch/utils/tensorboard/_pytorch_graph.py#L239
And will be updated soon.
